contrib/podmanimage/upstream/Dockerfile

```diff
@@ -49,23 +48,23 @@ RUN dnf -y install --exclude container-selinux \
      make BUILDTAGS="selinux seccomp"; \
      make install PREFIX=/usr; \
      cd /root/podman; \
-     git clone https://github.com/containers/conmon; \
+     git clone https://github.com/containers/conmon /root/podman/conmon; \
      cd conmon; \
      make; \
      install -D -m 755 bin/conmon /usr/libexec/podman/conmon; \
      git clone https://github.com/containernetworking/plugins.git $GOPATH/src/github.com/containernetworking/plugins; \
      cd $GOPATH/src/github.com/containernetworking/plugins; \
      ./build_linux.sh; \
      mkdir -p /usr/libexec/cni; \
-     cp bin/* /usr/libexec/cni; \
+     \cp -fR bin/* /usr/libexec/cni; \
```

Comment:
The backslash looks accidental, is it?

Reply:
Actually it's not accidental.  I was having issues getting the copy to do the force overwrite without asking for a confirming "Yes" without it.  If I remove it I get: 

```
STEP 4: RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /etc/containers/storage.conf
sed: can't read /etc/containers/storage.conf: No such file or directory
Error: error building at STEP "RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /etc/containers/storage.conf": error while running runtime: exit status 2
```

Reply:
I don't see those errors being related to the `\cp` line?

Reply:
Yeah I was scratching my head about that too.  But if I remove that "\" before cp, and only that, that's the error I get.  To try and figure out what was causing it initially, I ran all the commands by hand on a VM this weekend.  It wouldn't copy the files without a confirmation unless I added the slash.

It's failing later with this libpod.conf not found error as that's lower down in the same RUN statement and the libpod.conf file doesn't get created as the cp fails out prior.  So a step or two later when the sed happend on libpod.conf, this error gets thrown due to the prior failure.

Reply:
I am fine with the \cp, since this is just telling the script to ignore any aliases that have been setup in bashrc script.